flash/text/classification/data.py

```diff
@@ -82,6 +83,24 @@ def __init__(
             class_to_label_mapping[cls] = label
         self.set_state(ClassificationState(class_to_label_mapping))
 
+    @staticmethod
+    def version() -> str:
+        return "0.0.1"
+
+    def save_state_dict(self) -> Dict[str, Any]:
```

Comment:
this is not saving, right ? `get_state_dict` ?

Reply:
Done !